src/woo_publications/publications/admin.py

```diff
@@ -38,6 +43,85 @@
 )
 
 
+@admin.action(
+    description="Change %(verbose_name_plural)s owner(s)", permissions=["change"]
+)
+def change_owner(
+    modeladmin: PublicationAdmin | DocumentAdmin,
+    request: HttpRequest,
+    queryset: models.QuerySet[Publication | Document],
+):
+    assert isinstance(request.user, User)
+    model_name = str(model_ngettext(queryset))
+    opts = modeladmin.model._meta
+    form = ChangeOwnerForm(request.POST if request.POST.get("post") else None)
+
+    changeable_objects = [
+        format_html(
+            '<a href="{}">{}</a>',
+            reverse(
+                f"admin:{opts.app_label}_{opts.model_name}_change",
+                kwargs={"object_id": item.id},
+            ),
+            item.officiele_titel,
+        )
+        for item in queryset
+    ]
+
+    if (post := request.POST) and post.get("post"):
+        if form.is_valid():
+            if eigenaar_pk := post.get("eigenaar"):
+                owner = OrganisationMember.objects.get(pk=eigenaar_pk)
+            else:
+                owner = OrganisationMember.objects.create(
+                    identifier=post.get("identifier"), naam=post.get("naam")
+                )
+
+            for obj in queryset:
+                obj.eigenaar = owner
+                obj.save()
+
+                audit_admin_update(
+                    content_object=obj,
+                    object_data=serialize_instance(obj),
+                    django_user=request.user,
+                )
+
+            modeladmin.message_user(
+                request,
+                _("Successfully changed %(count)d owner(s) of %(items)s.")
+                % {
+                    "count": queryset.count(),
+                    "items": model_ngettext(modeladmin.opts, queryset.count()),
+                },
+                messages.SUCCESS,
+            )
+
+            modeladmin.model.objects.bulk_update(queryset, ["eigenaar"])
+            return
+
+    context = {
+        **modeladmin.admin_site.each_context(request),
+        "title": _("Change owner of multiple objects"),
```

Comment:
```suggestion
        "title": _("Change owner"),
```
shorter probably works?